db/index.js

```diff
@@ -0,0 +1,148 @@
+  picture: String, // img url
+  bio: String,
+  comments: [{ // form filled -> not authenticated
+    name: String,
+    comment: String
+  }]
+});
+
+const SuperlativePageSchema = mongoose.Schema({ // each superlative gets its own page
+  superlative: String, // superlative = most likely to be particular thing (ex. ax murderer)
+  nominees: [{ // form filled
+    name: String,
+    votes: Number
+  }]
+});
+
+const EventPageSchema = mongoose.Schema({
+  name: { type: String, require: true },
+  date: String, // or number (the number of milliseconds since the epoch)
+  description: String,
+  comments: [{
+    name: String,
+    comment: String
+  }]
+});
+
+const ShoutoutsPageSchema = mongoose.Schema({
+  category: { type: String, require: true }, // examples: quote, shoutouts to everyone, shoutout to staff, memes, etc.
+  text: String,
+  image: String, // url to img
+  name: String, // person saying this
+  target: String // the person this is about (ex.)
+});
+
+// ==================
+// ===== Models =====
+// ==================
+
+const StudentPage = mongoose.model('StudentPage', StudentPageSchema);
+const SuperlativePage = mongoose.model('SuperlativePage', SuperlativePageSchema);
+const EventPage = mongoose.model('EventPage', EventPageSchema);
+const ShoutoutsPage = mongoose.model('ShoutoutsPage', ShoutoutsPageSchema);
+
+// ============================
+// ===== Helper Functions =====
+// ============================
+
+// a helper function to use the correct model
+// returns a model instance with the data
+const getModel = (modelType, data) => {
+  if (modelType === 'student') {
+    return new StudentPage(data);
+  } else if (modelType === 'superlative') {
+    return new SuperlativePage(data);
+  } else if (modelType === 'event') {
+    return new EventPage(data);
+  } else if (modelType === 'shoutout') {
+    return new ShoutoutsPage(data);
+  } else {
+    console.log('ERROR! This is an unhandled case in the db -> index.js -> getModel');
+  }
+};
+
+// returns a the model for the functionality
+const getModelType = (modelType) => {
+  if (modelType === 'student') {
+    return StudentPage;
+  } else if (modelType === 'superlative') {
+    return SuperlativePage;
+  } else if (modelType === 'event') {
+    return EventPage;
+  } else if (modelType === 'shoutout') {
+    return ShoutoutsPage;
+  } else {
+  }
+};
+
+// ==============================
+// ===== Exported Functions =====
+// ==============================
+
+// ======================
+// === Save Functions ===
+// ======================
+
+const save = (modelType = 'student', data) => {
+  return getModel(modelType, data).save()
+    .then(function (model) {
+      return model;
+    })
+    .catch((e) => {
+      console.log('ERROR!\n  In db.save() when saving a model of type', modelType, '\n', e);
+    });
+};
+
+// ======================
+// === Load Functions ===
+// ======================
+
+// identifyier is an object that has the unique key/value pair
```

Comment:
Minor typo ("identifyier"). You can change it next time. Or not. But I've learned that you care about these things ;)